### PR TITLE
[ci skip] Fix instructions in README

### LINKED ISSUE #284: Clarify docs on node installation vs 'npm install'
Just now, looking at the [README](https://github.com/exercism/xecmascript/blob/master/README.md), the wording is unclear.  It might very well look as if typing 'npm install' is the command to install NodeJS itself, when actually it is the command to install the project dependencies, and should be run _after_ installing node on your machine.

screen shot 2017-05-09 at 5 58 27 pm

Success Criteria:
- the wording is expanded to show that installing NodeJS is the first step
- it is clearly explained that running `npm install` is for installing dependencies locally

### PR DESCRIPTION
**Pull request by tejasbubane**
Update instructions for installing dependencies and running tests.

Closes #284

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,24 +4,33 @@
...
 
+#### Test All Assignments
+The make script will test all exercises:
 
-### All Assignments
-The make script will build and test all exercises.
+    % make test
+
+#### Test Specific Assignment
+Pass exercise name to make script to run test for a specific exercise:
```
Comment (matthewmorgan) [inline README.md]: Maybe:

> Pass the exercise name to make script to run the tests for a specific exercise:

**Comment by tejasbubane** [comment]
@matthewmorgan Changed as per your request. Also the command for running single file was wrong. Fixed that as well.

**Comment by matthewmorgan** [comment]
@tejasbubane should we change this to reflect the changes you made in #291 -- using `make` instead of `make test`?

Or we could merge this now, since it will still work this way.  Let me know what you think, I'm fine either way.

**Comment by tejasbubane** [comment]
> Let me know what you think, I'm fine either way.

I think you can go ahead and merge this one, I will make the relevant change in that PR.

**Comment by matthewmorgan** [comment]
Thanks @tejasbubane!!

**Pull request merged**